packages/docs/src/styles/_layout.scss

```diff
@@ -40,6 +40,10 @@ Page layout elements
 .docs-root {
   background-color: $content-background-color;
 
+  // usually this element is _way_ longer than the viewport, but sometimes it isn't.
+  // this ensures that it'll always at least fill the viewport so white page doesn't show through.
```

Comment:
we can't because of the dark theme. `.pt-dark` is set on this element so that's where the colors have to be.